vmdb/app/models/automation_request.rb

```diff
@@ -2,14 +2,10 @@ class AutomationRequest < MiqRequest
   alias_attribute :automation_tasks, :miq_request_tasks
 
   TASK_DESCRIPTION  = 'Automation Request'
-  REQUEST_TYPES     = %w{ automation }
   DEFAULT_NAMESPACE = "SYSTEM"
   DEFAULT_CLASS     = "PROCESS"
   DEFAULT_INSTANCE  = "AUTOMATION_REQUEST"
 
-  validates_inclusion_of :request_type, :in => REQUEST_TYPES, :message => "should be #{REQUEST_TYPES.join(", ")}"
-  default_value_for(:requester)    { |r| r.get_user }
-
```

Comment:
Neither of these are intentional.  Message is just never initialized and `request_type` is passed to the create call in create_request but could be done with the default_value_for as well.